code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1053 +1,983 @@
+	if(!convertees.len)
+		fail_invoke()
+		log_game("Convert rune failed - no eligible convertees")
+		return
+	var/mob/living/new_cultist = pick(convertees)
+	if(!is_convertable_to_cult(new_cultist.mind) || new_cultist.null_rod_check())
+		user << "<span class='warning'>Something is shielding [new_cultist]'s mind!</span>"
+		fail_invoke()
+		log_game("Convert rune failed - convertee could not be converted")
+		return
+	var/time = 300
+	for(var/mob/living/M in orange(1,src))
+		if(iscultist(M))
+			time -= 100
+	time = Clamp(time, 0, 300)
+	if(time)
+		visible_message("<span class='warning'>[src] begins to glow with a sinister light...</span>")
+		for(var/mob/living/M in orange(1,src))
+			if(iscultist(M))
+				if(M != user)
+					M << "<span class='warning'>You have aided [user] in invoking the [cultist_name]. You may now safely depart.</span>"
+				else
+					M << "<span class='warning'>You begin the enlightenment of [new_cultist]. This will take [time / 10] seconds and you must stand still.</span>"
+		new_cultist << "<span class='userdanger'>Your head begins to ache...</span>"
+		if(!do_mob(user, new_cultist, time))
+			visible_message("<span class='warning'>[src] reluctantly falls dark.</span>")
+			new_cultist << "<span class='danger'>Your headache fades.</span>"
+			return
+	if(new_cultist == ticker.mode.sacrifice_target)
+		for(var/mob/living/M in orange(1,src))
+			if(iscultist(M))
+				M << "<span class='cult'>\"I desire this one for myself. <i>SACRIFICE THEM!</i>\"</span>"
+				fail_invoke()
+				log_game("Convert rune failed - convertee is sacrifice target")
+				return
+	new_cultist.visible_message("<span class='warning'>[new_cultist] writhes in pain as the markings below them glow a bloody red!</span>", \
+					  			"<span class='userdanger'><i>AAAAAAAAAAAAAA-</i></span>")
+	ticker.mode.add_cultist(new_cultist.mind)
+	new_cultist.mind.special_role = "Cultist"
+	new_cultist << "<span class='purple'><b><i>Your blood pulses. Your head throbs. The world goes red. All at once you are aware of a horrible, horrible, truth. The veil of reality has been ripped away \
+	and something evil takes root.</i></b></span>"
+	new_cultist << "<span class='purple'><b><i>Assist your new compatriots in their dark dealings. Your goal is theirs, and theirs is yours. You serve the Geometer above all else. Bring it back.\
+	</i></b></span>"
+
+
+//Rite of Tribute: Sacrifices a crew member to Nar-Sie. Places them into a soul shard if they're in their body.
+/obj/effect/rune/sacrifice
+	cultist_name = "Rite of Tribute"
+	cultist_desc = "Sacrifices a crew member to the Geometer. May place them into a soul shard if their spirit remains in their body."
+	icon_state = "3"
+	invocation = "Barhah hra zar'garis!"
+	color = rgb(255, 255, 255)
+	grammar = "veri nahlizet certum"
+
+/obj/effect/rune/sacrifice/invoke(mob/living/user)
+	var/turf/T = get_turf(src)
+	var/list/possible_targets = list()
+	for(var/mob/living/M in T.contents)
+		if(!iscultist(M))

```

Comment:
You need to also check that the cultist can participate in the invocation, i.e. !M.stat , or even !M.incapacitated() if you're okay with not letting cuffed cultist do invocations.